### PR TITLE
Implement barebones flask app

### PR DESCRIPTION
**Pull request by dodecaphonix**

### CODE REVIEW

File: routes.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
```
Comment (BlueSkies503) [inline routes.py:1]: What is flask?

Comment (dodecaphonix) [reply routes.py:1]: It's a python library that lets you spin up web servers.

**Review by BlueSkies503 — COMMENTED** [review]
lg2m

**Review by BlueSkies503 — APPROVED** [review]
lg2m

**Pull request merged**